ci/integration-tests-windows.groovy

```diff
@@ -72,13 +72,14 @@ node('py36') {
                                 make python; \
                                 python scripts/plugin/package_plugin.py; \
                                 cd python/lib/dcoscli; \
-                                python -c 'import urllib.request; urllib.request.urlretrieve(\\\"https://downloads.dcos.io/binaries/cli/windows/x86-64/latest/dcos.exe\\\", \\\"dist/dcos.exe\\\")'; \
-                                dist/dcos cluster remove --all; \
-                                dist/dcos cluster setup ${DCOS_TEST_URL} --insecure; \
-                                dist/dcos plugin add ../../../build/windows/dcos-core-cli.zip; \
-                                PATH=$PWD/dist:$PATH; \
                                 make env; \
-                                ./env/Scripts/tox -e py35-integration"
+                                rm -f ./env/Scripts/dcos.exe; \
+                                python -c 'import urllib.request; urllib.request.urlretrieve(\\\"https://downloads.dcos.io/cli/testing/binaries/dcos/windows/x86-64/master/dcos.exe\\\", \\\"dist/dcos.exe\\\")'; \
```

Comment:
not directly related to this PR, but we should also update this URL on Linux and mac

Reply:
I can do a PR for that now.